modules/cms-kit/src/Volo.CmsKit.MongoDB/Volo/CmsKit/MongoDB/Blogs/MongoBlogPostRepository.cs

```diff
@@ -118,6 +110,28 @@ public class MongoBlogPostRepository : MongoDbRepository<CmsKitMongoDbContext, B
                                     }).ToList();
     }
 
+    protected virtual async Task<List<Guid>> GetEntityIdsByTagId(Guid? tagId, CancellationToken cancellationToken)
+    {
+        var entityIdFilters = new List<Guid>();
+        if (!tagId.HasValue)
+        {
+            return entityIdFilters;
+        }
+
+        var entityIds =
+            await _entityTagManager.GetEntityIdsFilteredByTagAsync(tagId.Value, CurrentTenant.Id, cancellationToken);
```

Comment:
Why do we use a domain service in here, I don't think this is good. Domain Services are identical to implementing domain logic, for complex scenarios. But, here we just use it to get a list of `entityIds`. 

Also, it occurs to me we can remove the following two methods from the `EntityTagManager` class because they are only used to wrap repository methods:

https://github.com/abpframework/abp/blob/7e03f4b48a316e40c60e90f080c9e02113f1fe2a/modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Tags/EntityTagManager.cs#L75-L88

What do you think @enisn?

Reply:
Yeah, it seems there is no **domain logic** here, they only wrap repository methods and brings complexity

Reply:
I have created a separate issue for this, it can be better to do this change in a preview version. Because it introduces `breaking-changes`. 

issue: https://github.com/abpframework/abp/issues/17327